src/builtin/echo.c

```diff
@@ -0,0 +1,51 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   echo.c                                             :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: mhoyer <mhoyer@student.42.fr>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/07/06 07:37:23 by mhoyer            #+#    #+#             */
+/*   Updated: 2023/07/06 07:37:29 by mhoyer           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <stdio.h>
+
+int	ft_strcmp(char *to_cmp, char *model)
+{
+	int	i;
+
+	i = 0;
+	while (to_cmp[i] && model[i] && to_cmp[i] == model[i])
+		i++;
+	return (to_cmp[i] - model[i]);
+}
+
+void	echo(char **param)
```

Comment:
```suggestion
int echo(char **param)
```
je pense qu'il faut que les builtin return un int qui sera l'exit code de la commande